tests/integration/states/pip.py

```diff
@@ -64,6 +64,16 @@ def test_pip_installed_errors(self):
                 shutil.rmtree(venv_dir)
 
```

Comment:
In use [here](https://github.com/saltstack/salt/blob/develop/tests/integration/modules/useradd.py#L41).